db/models/jobSchema.js

```diff
@@ -7,7 +8,15 @@ const JobSchema = mongoose.Schema({
         type: Boolean,
         default: false
     },
-    images: [Buffer]
+    images: [Buffer],
```

Comment:
What is the purpose of this variable when we have two image fields in the `JobAnalysisSchema`

Reply:
Yea this is unused. I can clean this up